packages/core/src/codewhisperer/service/transformByQ/transformApiHandler.ts

```diff
@@ -385,40 +385,182 @@ export function getImageAsBase64(filePath: string) {
     return `data:image/svg+xml;base64,${fileContents}`
 }
 
+/*
+ * Given the icon name from core/resources/icons/aws/amazonq, get the appropriate icon according to the user's theme.
+ * ex. getIcon('transform-file') returns the 'transform-file-light.svg' icon if user has a light theme enabled,
+ * otherwise 'transform-file-dark.svg' is returned.
+ */
+export function getIcon(iconName: string) {
+    let iconPath = iconName
+    const themeColor = vscode.window.activeColorTheme.kind
+    if (themeColor === vscode.ColorThemeKind.Light || themeColor === vscode.ColorThemeKind.HighContrastLight) {
+        iconPath += '-light.svg'
+    } else {
+        iconPath += '-dark.svg'
+    }
+    return getImageAsBase64(globals.context.asAbsolutePath(path.join('resources', 'icons', 'aws', 'amazonq', iconPath)))
```

Comment:
literals never need to be joined
```suggestion
    return getImageAsBase64(globals.context.asAbsolutePath(path.join('resources/icons/aws/amazonq', iconPath)))
```